index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+  <body>
+    <header class="container">
+      <nav class="top-bar">
+        <div class="status">
+          <div id="time-container">
+            <img
+              id="time-icon"
+              src="images/Time Style.png"
+              alt="not available!"
+              class="time"
+            />
+          </div>
+          <div id="status-container">
+            <img
+              id="status-icon"
+              src="images/Group 3.png"
+              alt="not available!"
+              class="internet-wifi-charge"
+            />
+          </div>
+        </div>
```

Comment:
- [ ] As the last code reviewer suggested, please 🙏🏽 remove this section because it is just a reference in the Figma template for the mobile version, it shouldn't be implemented. 👇🏽 
![image](https://user-images.githubusercontent.com/58642949/192073812-0fb08df6-d878-4116-8cef-e48f81421f94.png)